backend/python/app/services/implementations/story_service.py

```diff
@@ -55,6 +55,23 @@ def create_story(self, story, content):
 
         return new_story
 
+    def get_stories_available_for_translation(self, user):
+        all_stories = []
+        approved_languages = user.approved_languages
+        for lang in approved_languages:
+            level = approved_languages[lang]
+            stories = (
+                Story.query.filter(Story.level <= level)
+                .filter(~Story.translated_languages.any(lang))
+                .all()
+            )
+
+            for story in stories:
+                if not story in all_stories:
+                    all_stories.append(story)
+
+        return all_stories
```

Comment:
Let's do something like
`return [story.to_dict(include_relationships=True) for story in all_stories]`
Its a good habit to return with to_dict incase models have any sensitive info